docs/reference-guides/rancher-security/rancher-v2.6-hardening-guides/rke1-self-assessment-guide-with-cis-v1.20-benchmark.md

```diff
@@ -0,0 +1,2762 @@
+```
+
+### 1.3.2 Ensure that the --profiling argument is set to false (Automated)
+
+
+**Result:** pass
+
+**Remediation:**
+Edit the Controller Manager pod specification file /etc/kubernetes/manifests/kube-controller-manager.yaml
+on the master node and set the below parameter.
+--profiling=false
+
+**Audit:**
+
+```bash
+/bin/ps -ef | grep kube-controller-manager | grep -v grep
+```
+
+**Expected Result**:
+
+```console
+'--profiling' is equal to 'false'
+```
+
+**Returned Value**:
+
+```console
+root 13538 13518 2 10:28 ? 00:00:12 kube-controller-manager --cluster-cidr=10.42.0.0/16 --root-ca-file=/etc/kubernetes/ssl/kube-ca.pem --configure-cloud-routes=false --service-account-private-key-file=/etc/kubernetes/ssl/kube-service-account-token-key.pem --allocate-node-cidrs=true --kubeconfig=/etc/kubernetes/ssl/kubecfg-kube-controller-manager.yaml --allow-untagged-cloud=true --leader-elect=true --profiling=false --cloud-provider= --service-cluster-ip-range=10.43.0.0/16 --pod-eviction-timeout=5m0s --v=2 --address=0.0.0.0 --enable-hostpath-provisioner=false --node-monitor-grace-period=40s --terminated-pod-gc-threshold=1000 --use-service-account-credentials=true
+```
+
+### 1.3.3 Ensure that the --use-service-account-credentials argument is set to true (Automated)
+
+
+**Result:** pass
+
+**Remediation:**
+Edit the Controller Manager pod specification file /etc/kubernetes/manifests/kube-controller-manager.yaml
+on the master node to set the below parameter.
+--use-service-account-credentials=true
+
+**Audit:**
+
+```bash
+/bin/ps -ef | grep kube-controller-manager | grep -v grep
+```
+
+**Expected Result**:
+
+```console
+'--use-service-account-credentials' is not equal to 'false'
+```
+
+**Returned Value**:
+
+```console
+root 13538 13518 2 10:28 ? 00:00:12 kube-controller-manager --cluster-cidr=10.42.0.0/16 --root-ca-file=/etc/kubernetes/ssl/kube-ca.pem --configure-cloud-routes=false --service-account-private-key-file=/etc/kubernetes/ssl/kube-service-account-token-key.pem --allocate-node-cidrs=true --kubeconfig=/etc/kubernetes/ssl/kubecfg-kube-controller-manager.yaml --allow-untagged-cloud=true --leader-elect=true --profiling=false --cloud-provider= --service-cluster-ip-range=10.43.0.0/16 --pod-eviction-timeout=5m0s --v=2 --address=0.0.0.0 --enable-hostpath-provisioner=false --node-monitor-grace-period=40s --terminated-pod-gc-threshold=1000 --use-service-account-credentials=true
+```
+
+### 1.3.4 Ensure that the --service-account-private-key-file argument is set as appropriate (Automated)
+
+
+**Result:** pass
+
+**Remediation:**
+Edit the Controller Manager pod specification file /etc/kubernetes/manifests/kube-controller-manager.yaml
+on the master node and set the --service-account-private-key-file parameter
+to the private key file for service accounts.
+--service-account-private-key-file=<filename>

```

Comment:
```suggestion
`--service-account-private-key-file=<filename>`

```

Converting to inline code since Docusaurus will throw an error because it treats `<filename>` as an HTML tag.

Reply:
resolved